pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -123,14 +123,14 @@ type GrpcPodConfig struct {
 	// SecurityContextConfig can be one of `legacy` or `restricted`. The CatalogSource's pod is either injected with
 	// the right pod.spec.securityContext and pod.spec.container[*].securityContext values to allow the pod to run in
 	// Pod Security Admission(PSA) controller's `restricted` mode, or doesn't set these values at all, in which case the pod
-	// can only be run in PSA `baseline` or `privileged` namespaces. By default, SecurityContextConfig is set to `restricted`.
-	// If the value is unspecified, the default value of `restricted` is used.  Specifying any other value will result in a
+	// can only be run in PSA `baseline` or `privileged` namespaces. By default, SecurityContextConfig is set to `legacy`.
+	// If the value is unspecified, the default value of `legacy` is used.  Specifying any other value will result in a
```

Comment:
We should probably also add that "the default will be switched to `restricted` very soon and if you are a catalog author then please ensure that your next catalog image release is compatible with the `restricted` mode"

Reply:
Done!